packages/components/atoms/f-image-tile/src/components/ImageTile.vue

```diff
@@ -12,20 +47,108 @@ export default {
     name: 'ImageTile',
     components: {},
     props: {
+        href: {
+            type: String,
+            default: 'https://www.google.com'
```

Comment:
```suggestion
            default: null
```

as far as I understand on searchweb we don't have links for image tiles. In this case it's better to have null as a default value.